sample-app/src/components/InputDropdown.tsx

```diff
@@ -157,8 +160,8 @@ export default function InputDropdown({
   });
 
   function handleInputElementKeydown(evt: KeyboardEvent<HTMLInputElement>) {
-    if (evt.key === 'Enter') {
-      onInputChange(inputValue);
+    if (evt.key === 'Enter' && focusedSectionIndex === undefined && !onlySubmitOnOption) {
```

Comment:
is `focusedSectionIndex === undefined` needed? this is only trigger by onKeyDown in input element so would `focusedSectionIndex` ever be defined if we are in input element?